### ISSUE TITLE
Package scalable logos for platform use

### ISSUE THREAD

**Issue by janetpkr** [creativecommons/platform-initiative#19]
Create resizable logos for platform use as part of communications and developer's kits

**Comment by mattl** [comment]
Resizable logos of what? CC license icons and trademarks? We already have those.

**Comment by janetpkr** [comment]
Renamed to "scalable" logos. We offer standard sizes at http://creativecommons.org/about/downloads/, but I was talking to some product people at Aspiration Summit and they said that it's more up to date to provide scalable versions of our logos so that a platform can resize them easily to fit how they display them on the platform. Scalable vector format... does that make sense?

**Comment by mattl** [comment]
yep. We provide those from that page too.... SVG and EPS files.

I do think that page needs some love though.

---

Matt Lee
Creative Commons
Boston, MA, USA

**Comment by janetpkr** [comment]
Oh ok, so we have them. So I changed issue to "Package" it up nicely for platform use. I'll assign myself that and I can work on it with you down the line.

**Comment by mattl** [comment]
Yeah, @jayaw mentioned the idea of a press-pack for CC, I think we'd do this work for that too.
